addons/web/static/src/core/ui/block_ui.scss

```diff
@@ -1,8 +1,5 @@
 .o_blockUI {
 	cursor: wait;
-	opacity: 0.6;
-	
-	img {
-		animation: fa-spin 1s infinite steps(12);
-	}
+	-webkit-backdrop-filter: blur(5px);
```

Comment:
@stefanorigano 👋 
The `backdrop-filter` works on all modern browsers except Firefox. I think it’s worth keeping because it doesn’t break the layout if it’s not rendered and gives something extra when it is.
What do you think ?

Reply:
ok